wordpress-mobile/WordPress-Android
# Rename additionalInformation to footnote

Parent #13328 & #13329

This PR refactors `additionalInformation` to `footnote` in states, builders, adapters, layouts, styles, and string This name change more clearly aligns the type with the design making it easier for future developers to understand it place in the view.

To test:
- Run the backup process to the end
- Run the restore process to the end
- Note that everything works as expected, no funky crashes related to missing resources

PR submission checklist:
